### PR TITLE
style-guide: refresh layout info

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: contributing-guides/style-guide.md [hunk 1]
```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
 
 The basic format of each page should match the following template:
 
-```
-# command-name
+```md
+# <command-name>
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:10]: I don't like this, people will assume it needs to be wrapped in them, leading to just more linting and syntax fixing.

Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md]: Fixed: angle brackets are removed.

File: contributing-guides/style-guide.md [hunk 2]
```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
...
 
 > Short, snappy description.
 > Preferably one line; two are acceptable if necessary.
-> More information: <https://example.com>.
+> More information: <https://example.com/command-name/man/page>.
 
 - Example description:
 
-`command -opt1 -opt2 -arg1 {{arg_value}}`
+`command-name -opt1 -opt2 -arg1 {{arg1_value}}`
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:18]:
```suggestion
`command_name -opt1 -opt2 -arg1 {{arg1_value}}`
```

I prefer snake_case, lets see what others think before committing this

File: contributing-guides/style-guide.md [hunk 3]
```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
...
 
 - Example description:
 
-`command -opt1 -opt2`
+`command-name -opt1 -opt2`
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:22]:
```suggestion
`command_name -opt1 -opt2`
```

File: contributing-guides/style-guide.md [hunk 4]
```diff
@@ -36,10 +36,25 @@ For other ways to use `tldr-lint`, such as linting an entire directory, check ou
...
 Your client may be able to preview a page locally using the `--render` flag:
 
-```
+```sh
 tldr --render {{page.md}}
 ```
 
+### Aliases
+
+If some command is an alias for another then you can provide for it such Tl;Dr page:
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:45]:
```suggestion
If two or more commands are the same, alias pages can be created to provide the correct page for the user.
```

Comment (CleanMachine1) [reply contributing-guides/style-guide.md:45]: Here's another suggestion:

```suggestion
If two or more commands are the same, alias pages can be created to provide the name of the correct page for the user.
```

File: contributing-guides/style-guide.md (hunk 2 above)
Comment (CleanMachine1) [reply contributing-guides/style-guide.md:18]: Also it is used later in the command on this line.

File: contributing-guides/style-guide.md [hunk 5]
```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
...
 The basic format of each page should match the following template:
 
-```
+```md
 # command-name
 
 > Short, snappy description.
 > Preferably one line; two are acceptable if necessary.
-> More information: <https://example.com>.
+> More information: <https://example.com/command-name/man/page>.
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:14]:
```suggestion
> More information: <https://example.com>.
```
Often, if possible, links are to resources relating, instead of a manual page, and your suggestion promotes it which I personally do not like.

**Comment by CleanMachine1** [comment]
Isn't this PR just a smaller version of  https://github.com/tldr-pages/tldr/pull/7441/files

**Comment by EmilySeville7cfg** [comment]
> Isn't this PR just a smaller version of https://github.com/tldr-pages/tldr/pull/7441/files

It is intended to be just a part of #7441 (I thought it's better to split it) but now it has some enhancements that differs it from another PR.

File: contributing-guides/style-guide.md (hunk 5 above)
Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md]: I've replaced it with: `https://example.com/command-name/help/page`. And added a note:

```
> :bulb: Help page is an any documentation page here, not just man page.
```

Comment (sbrl) [reply contributing-guides/style-guide.md]: Generally we prefer, in order or preference:

1. Specific official documentation on the command-line program in question
2. The project's homepage
3. A link to a man page

File: contributing-guides/style-guide.md [hunk 6]
```diff
@@ -4,29 +4,36 @@ This page lists specific formatting instructions for `tldr` pages.
...
 
-The basic format of each page should match the following template:
+> :bulb: Placeholder syntax used in this document: `{{tip}}`.
 
-```
-# command-name
+The basic format of each page should match the following template and have at most 8 code examples:
+
+```md
+# {{command_name}}
```
Comment (sbrl) [inline contributing-guides/style-guide.md:12]: The token syntax isn't meant to be used here. Same goes for all the other places it's used outside of code blocks.

File: contributing-guides/style-guide.md [hunk 7]
```diff
@@ -96,9 +118,9 @@ Use backticks on the following:
 
 ## Imperative Mood
 
-Example descriptions have to be phrased in imperative mood.  
-For example, use `List all files`, instead of `Listing all files` or `File listing`.  
-This also applies to all translations by default, unless this is not possible for some reason.  
+Example descriptions have to be phrased in imperative mood.
```
Comment (sbrl) [inline contributing-guides/style-guide.md:121]: The double spaces at the end of each line here force markdown renders to put text on the next line - just like `<br />` in HTML. Let's keep this as-is. Same goes for all other instances below.

File: contributing-guides/style-guide.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md]: Angle braces are just removed.

File: contributing-guides/style-guide.md (hunk 7 above)
Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md:172]: It's due to the `.editorconfig` setting:

```
trim_trailing_whitespace = true
```

File: contributing-guides/style-guide.md [hunk 8]
```diff
@@ -4,40 +4,113 @@ This page lists specific formatting instructions for `tldr` pages.
...
 
-- Example description:
+- Start in a fullscreen mode:
 
-`command -opt1 -opt2`
+`krita --fullscreen`
 ```
 
+> :bulb: Help page is an any documentation/project/tutorial page here, not just man page.
+> Docs are preffered.
```
Comment (bl-ue) [inline contributing-guides/style-guide.md:69]:
```suggestion
> :bulb: Help page can be any documentation/project/tutorial page, not just a man page.
> Documentation pages are preferred.
```

**Comment by bl-ue** [comment]
@EmilySeville7cfg Will you please change `it takes more attention to copywriting` to `more attention must be paid to copywriting` on [line 201](https://github.com/tldr-pages/tldr/pull/7951/files#diff-ba70367725869e0ef89b620cf11a6e37c851a02b88aed69515ed2b9c75534ebeR201)? (I'm not able to submit a suggestion.)

**Comment by EmilySeville7cfg** [comment]
Yeah, I am going to do it as soon as possible 😁

File: contributing-guides/style-guide.md [hunk 9]
```diff
@@ -4,38 +4,77 @@ This page lists specific formatting instructions for `tldr` pages.
...
+
+- Start in a fullscreen mode:
+
+`krita --fullscreen`
+```
+
+> :bulb: Help page can be any documentation/project/tutorial page, not just a man page.
+> Documentation pages are preferred.
 
 There actually is a linter/formatter that enforces the format above.
```
Comment (sbrl) [inline contributing-guides/style-guide.md:71]: Perhaps this will read better?
```suggestion
There is a linter that enforces the format above.
```

**Review by sbrl — APPROVED** [review]
Looks good to me! Just 1 teeny correction to add to this PR.

File: contributing-guides/style-guide.md [hunk 10]
```diff
@@ -4,40 +4,113 @@ This page lists specific formatting instructions for `tldr` pages.
...
+> :bulb: Placeholder syntax is not used in this chapter, but instead of:
+>
+> - `command_name`
+> - `https://example.com/command-name/help/page`
+> - `Code description`
+> - `first_options`
+> - `second_options`
+> - `page_name`
+>
+> you can use any valid value as they are placeholders.
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:16]: I am confused as to what this part even is, could you explain

File: contributing-guides/style-guide.md [hunk 11]
```diff
@@ -4,40 +4,113 @@ This page lists specific formatting instructions for `tldr` pages.
...
+
+The basic format of each page should match the following template and have at most 8 code examples:
```
Comment (CleanMachine1) [inline contributing-guides/style-guide.md:18]:
```suggestion
The basic format of each page should match the following template and have at most 8 command examples:
```

File: contributing-guides/style-guide.md (hunk 10 above)
Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md:16]: It is a note about that `{{`, `}}` are not used here explicitly in some cases.

File: contributing-guides/style-guide.md [hunk 12]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
 
 ## Layout
 
-The basic format of each page should match the following template:
+The basic format of each page should match the following template and have at most 8 command examples:
 
-```
-# command-name
+```md
+# command_name
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:10]: For the actual pages, we use a space, so why not here?
```suggestion
# command name
```

File: contributing-guides/style-guide.md [hunk 13]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
...
 
-`command -opt1 -opt2`
+- Start in a fullscreen mode:
+
+`krita --fullscreen`
 ```
 
-There actually is a linter/formatter that enforces the format above.
+> :bulb: Help page can be any documentation/project/tutorial page, not just a man page.
+> Documentation pages are preferred.
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:58]: How about this?
```suggestion
> :bulb: The help page can be any documentation/project/tutorial page, not just a man page,
> but documentation pages are preferred.
```

File: contributing-guides/style-guide.md [hunk 14]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
...
+
+There is a linter that enforces the format above.
 It is run automatically on every pull request,
 but you may install it to test your contributions locally before submitting them:
 
-```
+```sh
 npm install --global tldr-lint
-tldr-lint {{page.md}}
+tldr-lint page_name.md
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:66]: This sounds a little more tldr-ish:
```suggestion
tldr-lint path/to/tldr_page.md
```

File: contributing-guides/style-guide.md [hunk 15]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
...
 ```
 
 For other ways to use `tldr-lint`, such as linting an entire directory, check out (what else!)
 [`tldr tldr-lint`](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr-lint.md). Alternatively, you can also use its alias `tldrl`.
 
 Your client may be able to preview a page locally using the `--render` flag:
 
+```sh
+tldr --render page_name.md
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:75]: Same here:
```suggestion
tldr --render path/to/tldr_page.md
```

File: contributing-guides/style-guide.md [hunk 16]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
...
 ```
-tldr --render {{page.md}}
+
+### Aliases
+
+If two or more commands are the same, alias pages can be created to provide the name of the correct page for the user.
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:80]:
```suggestion
If a command can be called with alternative names (like `vim' can be called by `vi`), alias pages can be created to point the user to the original command name.
```

File: contributing-guides/style-guide.md [hunk 17]
```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
...
+Example:
+
+```md
+# vi
+
+> This command is an alias of `vim`.
+
+- View documentation for the original command:
+
+`tldr vim`
```
Comment (marchersimon) [inline contributing-guides/style-guide.md:102]:
```suggestion
`tldr vim`

Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
```

File: contributing-guides/style-guide.md (hunk 12 above)
Comment (sbrl) [reply contributing-guides/style-guide.md:10]: Isn't the title supposed to match the command's name?

Comment (marchersimon) [reply contributing-guides/style-guide.md:10]: Sure, but we do write `# git commit`, instead of `# git_commit`. But actually, this change doesn't really matter.

**Review by marchersimon — APPROVED** [review]
Looks good

**Review by blueskyson — APPROVED** [review]

File: contributing-guides/style-guide.md (hunk 12 above)
Comment (sbrl) [reply contributing-guides/style-guide.md:10]: * `git-commit.md` becomes `# git commit` but yes.

**Review by sbrl — APPROVED** [review]
